src/Core/ServiceWrapper/CLI/CliOption.cs

```diff
@@ -0,0 +1,65 @@
+﻿using CommandLine;
+using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using winsw.Native;
+using WMI;
+
+namespace winsw.CLI
+{
+    public abstract class CliOption
+    {
+        [Option("configFile", HelpText = "Configurations File")]
+        public string ConfigFile { get; set; }
+
+        [Option("elevated", HelpText = "Elevated Command Prompt", Default = false)]
+        public bool Elevated { get; set; }
+
+        [Option("redirect", HelpText = "Redirect Logs")]
```

Comment:
:warning: Breaking change: `CommandLineParser` is not going to support forward slashes.

Reply:
This argument has been deprecated. We don't need a new one for it.

Reply:
I think this deprecating mean we just don't allowing users to elevate manually and still we elevate internally before execute the command? So is this update to Program class works?
```c#
            if (Environment.OSVersion.Version.Major == 5)
            {
                // Windows XP
                elevated = true;
            }
            else
            {
                elevated = IsProcessElevated();
            }
```